am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -68,27 +71,36 @@ public void revokeResourceAccess(ExplicitAccessMetadata explicitAccessMetadata)
      * @param userId       (accessorId)
      * @param resourceId   resource id
      * @param resourceJson json
-     * @return resourceJson or null
+     * @return filtered resourceJson based on READ permissions or null if no READ permissions on resource
```

Comment:
Better but result is not just `resourceJson`. It is whole envelope with resource ID, filtered JSON and map of permissions. Please update.

Reply:
👍

Reply:
👍